requirements.txt

```diff
@@ -1,3 +1,3 @@
-sphinx==2.2.0
+sphinx==3.2.1
```

Comment:
As it is now, we're not testing against the current Sphinx version at all. Ideally, we'd test against a couple of Sphinx versions. Since implementing that is a sizeable project on its own, I figured we should preferably test against the current version. We can drop the `==3.2.1` to always use the latest version.

Reply:
Doing `sphinx>=2` would install the latest version I'm pretty sure, so it would work.